lib/eunit/doc/overview.edoc

```diff
@@ -994,7 +994,7 @@ specified node. `local' means that the current process will handle both
 setup/teardown and running the tests - the drawback is that if a test
 times out so that the process is killed, the <em>cleanup will not be
 performed</em>; hence, avoid this for persistent fixtures such as file
-operations. In general, 'local' should only be used when:
+operations. In general, `local` should only be used when:
```

Comment:
This is wrong. The second single quote should **not** be changed. Should be:

    `local'

Please amend your commit.